packages/core/core/src/RequestGraph.js

```diff
@@ -150,6 +153,9 @@ export default class RequestGraph extends Graph<RequestGraphNode> {
   async transform(request: AssetRequest) {
     try {
       let start = Date.now();
+      if (!(await realpath(request.filePath)).includes('node_modules')) {
```

Comment:
Maybe do this check inside the validator runner?

Reply:
Sure will update it